packages/gatsby-theme-newrelic/gatsby-config.js

```diff
@@ -45,19 +39,6 @@ module.exports = ({
           path: 'src/announcements',
         },
       },
-      {
-        resolve: 'gatsby-plugin-gdpr-tracking',
```

Comment:
[nit] Since we no longer use this plugin, we can also remove it from `package.json` so that it is no longer installed.